imap_processing/codice/codice_l0.py

```diff
@@ -37,9 +36,20 @@ def decom_packets(packet_file: Path) -> dict[int, xr.Dataset]:
     packets : dict[int, xarray.Dataset]
         Mapping from apid to ``xarray`` dataset, one dataset per apid.
     """
+    # TODO: Currently need to use the 'old' packet definition for housekeeping
+    #       because the test housekeeping packet has LAST_OPCODE=0, and the
+    #       telemetry definition "STATES" tab doesn't allow for a 0 value for
+    #       this. Need to figure out if the test packet is bad, or the telemetry
+    #       definition is wrong.
```

Comment:
This is an interesting case. Does an Enumeration need to exist for every possible value or should `space_packet_parser` just return the raw_value here and not decode anything rather than raising an error? Gavin might have some insight here as well.

Reply:
From what I understand from Gavin's comment in the `xtce` slack channel, it sounds like his preference would be to explicitly define what a value of `0` means in the packet definition file. I will ask Joey about this in our next tagup.